rules/60-block-scheduler.rules

```diff
@@ -0,0 +1,3 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="add|change", KERNEL=="sd*[!0-9]|sr*", ATTR{queue/scheduler}="bfq"
```

Comment:
there should be a comment here I figure, explaining the situation briefly. (And I'd really clarify that this is a technical solution for a political problem in the kernel community and that we believe this shouldn't be here).

I think this should also carry a SUBSYSTEM="block" check, no?

Also, needs a NEWS entry

Reply:
Duh, I moved this to a separate file and forgot to copy all the "headers". So this would need to get fixed.